samples/redis/sample.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Basic Redis usage
+summary: Passing data between endpoints using Redis
+component: Core
+reviewed: 2024-09-12
+---
+
+This sample shows how to pass large data between endpoints using [Redis](https://redis.io/).
```

Comment:
I am unclear why "large data" is essential here, is this something customers mentioned before, to use Redis instead of DataBus?

Reply:
I think we should add some context here that usually we recommend to users here to use the Databus, with full knowledge that Redis is not supported.

This shows a sample of how to achieve something similar, but the platform does not support it out of the box. 

I believe if we make that clear to the user here, that they would be more likely to reach out to us and share their use cases